exercises/practice/pig-latin/build.gradle

```diff
@@ -22,3 +22,7 @@ test {
     events = ["passed", "failed", "skipped"]
   }
 }
+
+tasks.named("compileTestJava").configure {
+  dependsOn("copyTestsFilteringIgnores")
+}
```

Comment:
This change should no longer be needed, as the Gradle configuration was fixed in #2339